Travel Scheduler/View Controllers/ScheduleViewController.m

```diff
@@ -0,0 +1,193 @@
+//
+//  ScheduleViewController.m
+//  Travel Scheduler
+//
+//  Created by aliu18 on 7/18/19.
+//  Copyright © 2019 aliu18. All rights reserved.
+//
+
+#import "ScheduleViewController.h"
+#import "TravelSchedulerHelper.h"
+#import "DateCell.h"
+
+@interface ScheduleViewController () <UICollectionViewDelegate, UICollectionViewDataSource, DateCellDelegate>
+
+@property (strong, nonatomic) UILabel *header;
+@property (strong, nonatomic) UICollectionView *collectionView;
+@property (strong, nonatomic) UIScrollView *scrollView;
+@property (strong, nonatomic) NSDate *startDate;
+@property (strong, nonatomic) NSDate *endDate;
+@property (strong, nonatomic) NSMutableArray *dates;
+@property (strong, nonatomic) NSDate *selectedDate;
+@property (nonatomic) int numHours;
+
+@end
+
+static int startY = 35;
```

Comment:
Really liked the static objects. I saw that this was handy when specifying the CGFRame.